docu/Concepts/Snippets/Authorization/concept.md

```diff
@@ -0,0 +1,28 @@
+# Authorization and Private Keys
+## Keys
+For creating transactions ed25519 keys are used for signing. 
+As long the user is the only controlling the private key he is the only one
+how can sign transactions on his behalf. 
+It is a core concept of all crypto currencies and important for the concept,
+that the user has full control over his data. 
+
+Usually crypto currencies like bitcoin or iota save the keys on local system,
+maybe additional protected with a password which is used to encrypt the keys.
+
+## Gradido
+Gradido should be easy to use, so we must offer a solution for everyone not that fit
+with computer, as easy to use like paypal.
+For that role we have the Login-Server.
+It stores the private keys of the user encrypted with there email and password. 
+Additional it stores the passphrase which can be used to generate the private key,
+encryted with server admin public key. So only the server admin can access the keys
+with his private key. [not done yet]
```

Comment:
Was bedeutet `not done yet` in diesem Zusammenhang? Ich dachte es wäre möglich days Passwort des Nutzers zurückzusetzen bzw zu verändern. D.h. der Armin kann den Private key wiederherstellen?

Reply:
Momentan läuft das Passwort zurücksetzen automatisch, die Passphrase wird unverschlüsselt gespeichert. 
Was noch nicht fertig ist, das die Passphrase noch nicht mit dem Admin Key verschlüsselt gespeichert wird. 
Das verschlüsselt Speichern ist dabei das einfachste. 
Aber wir brauchen dann noch ein entsprechendes Interface.
Wenn ein Benutzer ein Passwort Reset anfordert, muss der Admin benachrichtigt werden, sich einloggen, sein Privat Key eingeben und den Passwort Reset authorisieren. Dann bekommt der Benutzer seine Passwort Reset E-Email. 
Dann muss die entschlüsselte Passphrase oder der Private Key des Users so lange im Speicher behalten werden, bis der User auf seinen Link geklickt hat und sein Private key mit seinem neuen Passwort wieder verschlüsselt werden konnte.